wiki/wiki_entry_moritz_trabold.qmd

```diff
@@ -0,0 +1,45 @@
+---
+title: "Conversations about the elections on Twitter: Towards a structural understanding of Twitter's relation with the political and the media field"
+subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+author: "Moritz Trabold"
+categories:
+  - "Media Psychology"
+  - "Digital Media"
+  - "Well Being"
+---
+
+## K: Key Quote
+"In general, we can state that variations in symbolic capital (i.e. legitimation), which we define as the number of mentions and replies one receives, are related to the positions the agents hold as politician, media or citizen agent. In addition, the variance in conversation practices on Twitter is significantly related to the different user types. Below, we wrap up the main findings and elaborate upon future theoretical endeavours and the limitations of the study." (D´heer & Verdegem, 2014, p. 731)
+
+
+## A: Argument
+The main argument of the text is that Twitter has become an important platform for political communication and its role in shaping the political and media fields should be studied from a structural perspective. The authors argue that the interactions on Twitter during the elections reflect broader power relations and structures in society, which can be studied through network analysis and discourse analysis.
+
+The main argument of the text is that Twitter has become an important platform for political communication and its role in the shaping of the political and media fields should be studied from a structural perspective. The authors argue that network and discourse analysis can be used to study how interactions on Twitter during the election reflect broader power relations and structures in society.
+
+
+## R: Relationship
+This argument is in line with the general theoretical argument that social media platforms are not neutral platforms, but are shaped by and shape social structures and power relations. The authors also draw on theories of the public sphere and media ecology to argue that Twitter has become a new space for political discourse. However, its influence should be studied in relation to traditional media and other social structures.
+
+As a whole, the text highlights the importance of adopting a structural perspective in the study of the role of Twitter and other social media platforms in politics and the media. It also emphasises the need for interdisciplinary approaches combining communication, political science, sociology and other theories.
+
+
+## Q:
+1. Did the study take into account the potential impact of Twitter bots and fake accounts on the political conversations that took place during the elections?
+2. How do demographic characteristics of Twitter users, such as age or gender, influence how they engage in political conversation during elections?
+
+
```

Comment:
How does this all relate to autonomy or individuals' autonomy need experiences?

Reply:
I changed the article because the one I used before didn't fit the main question "What constitutes autonomy in digital communication?"